packages/flutter/lib/src/material/chip.dart

```diff
@@ -1628,15 +1603,15 @@ class _ChipRenderTheme {
       return false;
     }
     final _ChipRenderTheme typedOther = other;
-    return typedOther.avatar == avatar
-        && typedOther.label == label
-        && typedOther.deleteIcon == deleteIcon
-        && typedOther.brightness == brightness
-        && typedOther.padding == padding
-        && typedOther.labelPadding == labelPadding
-        && typedOther.showAvatar == showAvatar
-        && typedOther.showCheckmark == showCheckmark
-        && typedOther.canTapBody == canTapBody;
+    return typedOther.avatar == avatar &&
+        typedOther.label == label &&
+        typedOther.deleteIcon == deleteIcon &&
+        typedOther.brightness == brightness &&
+        typedOther.padding == padding &&
+        typedOther.labelPadding == labelPadding &&
+        typedOther.showAvatar == showAvatar &&
+        typedOther.showCheckmark == showCheckmark &&
+        typedOther.canTapBody == canTapBody;
```

Comment:
old style was more readable

Reply:
ACK.  Sorry, I hit the 'dartfmt' key, and forgot to go back and revert the spurious changes that don't match Flutter style.  Most of the changes in this file were due to that: I've reverted them.

Reply:
fwiw i'm agnostic to the ones i didn't comment on (most of them)